Documentation/AdvancedUsage.md

```diff
@@ -1285,7 +1285,7 @@ By default, `DataTask` and `DownloadTask` values do not cancel the underlying re
 ```swift
 let request = AF.request(...) // Creates the DataRequest.
 let task = Task { // Produces a `Task<DataResponse<TestResponse, AFError>, Never> value.
-    await request.serializingDecodable(TestResponse.self, automaticallyCancelling: true).response
+    await request.serializingDecodable(TestResponse.self, automaticallyCancelling: false).response
```

Comment:
```suggestion
    await request.serializingDecodable(TestResponse.self).response
```

It's probably better to just remove the explicit parameter so readers don't think it's required.

Reply:
Thanks!